store/root/store.go

```diff
@@ -290,7 +318,12 @@ func (s *Store) Commit(cs *store.Changeset) ([]byte, error) {
 
 	// commit SS async
 	eg.Go(func() error {
-		if err := s.stateStore.ApplyChangeset(version, cs); err != nil {
+		// if we're migrating, we don't want to commit to the state storage
```

Comment:
>		// if we're migrating, we don't want to commit to the state storage
Why not? How will a node read latest state? Unmarshal the entire changeset at latest version then fetch the key?  Why is that preferable to just writing the version to SS in the first place?

I also don't see anywhere where we set `s.stateStorage = nil` as a flag to indicate a migration is in progress?